Validation/HGCalValidation/plugins/HGCalTriggerValidator.cc

```diff
@@ -194,15 +203,16 @@ void HGCalTriggerValidator::bookHistograms(DQMStore::IBooker &iBooker,
 }
 
 void HGCalTriggerValidator::dqmAnalyze(edm::Event const &iEvent,
-                                       edm::EventSetup const &iSetup,
+                                       const edm::EventSetup &iSetup,
                                        Histograms const &histograms) const {
   int tc_n = 0;
   int cl_n = 0;
   int cl3d_n = 0;
   int tower_n = 0;
 
-  triggerTools_->eventSetup(iSetup);
-
+  auto const triggerGeometry_ = iSetup.getHandle(triggerGeomToken_);
```

Comment:
The trailing underscore should be reserved for class members. `triggerGeometry_` should be `triggerGeometry`.